jsconfig.json

```diff
@@ -1,11 +1,13 @@
 {
   "compilerOptions": {
     "target": "ESNext",
-    "module": "ESNext",
+    "module": "nodenext",
```

Comment:
VSCode TS tells, that if "target" is "ESNext" then "module" needs to be "nodenext"